commons-dlp/src/main/resources/conf/portal/configuration.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<!--
+
+    Copyright (C) 2020 eXo Platform SAS.
+
+    This is free software; you can redistribute it and/or modify it
+    under the terms of the GNU Lesser General Public License as
+    published by the Free Software Foundation; either version 2.1 of
+    the License, or (at your option) any later version.
+
+    This software is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+    Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public
+    License along with this software; if not, write to the Free
+    Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+    02110-1301 USA, or see the FSF site: http://www.fsf.org.
+
+-->
+
+<configuration
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.exoplatform.org/xml/ns/kernel_1_2.xsd http://www.exoplatform.org/xml/ns/kernel_1_2.xsd"
+    xmlns="http://www.exoplatform.org/xml/ns/kernel_1_2.xsd">
+  
+  <component>
+    <key>org.exoplatform.commons.dlp.queue.QueueDlpService</key>
+    <type>org.exoplatform.commons.dlp.queue.impl.QueueDlpServiceImpl</type>
+  </component>
+  
+  <component>
+    <key>org.exoplatform.commons.dlp.dao.DlpOperationDAO</key>
+    <type>org.exoplatform.commons.dlp.dao.impl.DlpOperationDAOImpl</type>
+  </component>
+  
+  <component>
+    <key>org.exoplatform.commons.dlp.processor.DlpOperationProcessor</key>
+    <type>org.exoplatform.commons.dlp.processor.impl.DlpOperationProcessorImpl</type>
+  </component>
+  
+  <external-component-plugins>
+    <target-component>org.exoplatform.services.scheduler.JobSchedulerService</target-component>
+    <component-plugin>
+      <name>DlpJob</name>
+      <set-method>addPeriodJob</set-method>
+      <type>org.exoplatform.services.scheduler.PeriodJob</type>
+      <description>Init dlp job</description>
+      <init-params>
+        <properties-param>
+          <name>job.info</name>
+          <description>details of the execution of the dlp job</description>
+          <property name="jobName" value="Dlp"/>
+          <property name="groupName" value="Dlp"/>
+          <property name="job" value="org.exoplatform.commons.dlp.job.DlpJob"/>
+          <property name="repeatCount" value="0"/>
+          <property name="period" value="${exo.dlp.job.period:5000}"/><!-- default 5 seconds -->
+          <property name="startTime" value="+6000"/><!-- start after 1 minute delay -->
+          <property name="endTime" value=""/>
+        </properties-param>
+      </init-params>
+    </component-plugin>
+  </external-component-plugins>
```

Comment:
CronJob or PeriodJob ?

With a cronJob, we can say to Quartz to not start a new execution if the first one is not finished

In addition, I thinkg that execution time each 1 minute is sufficient